### ISSUE TITLE
to_string is used instead of tostring

### ISSUE THREAD

**Issue by nulman** [Kamal-Sadek/Liberal-Crime-Squad#9]
in some files, such as basemode/activate.cpp and title/titlescreen.cpp
to_string is used instead of tostring
this causes compilation failure (at least on windows using codeblocks)

**Comment by Kamal-Sadek** [comment]
Yeah, this is an issue that exists because of moving to C++11. Before that, the project used its own version of tostring (as far as I remember), but now that we are officially ported to C++11 we should remove all the functions that were "hacked in", such as this tostring.
